content/spin/see-what-people-have-built-with-spin.md

```diff
@@ -40,6 +58,8 @@ The 'Finicky Whiskers' series covers the journey of creating a game with an arch
 * [Part 3: The Microservices](https://www.fermyon.com/blog/finicky-whiskers-part-3-microservices)
 * [Part 4: Spin, Containers, Nomad, and Infrastructure](https://www.fermyon.com/blog/finicky-whiskers-part-4-infrastructure)
 
+## Accessing External APIs
```

Comment:
Looking at this, it's really a demo of making an outbound HTTP call, not really a "see cool things people have built" thing.

Reply:
Great idea, this is added in fresh commit.

Reply:
I might reword this, as apposed to removing it, because it is a cool feature.

Reply:
Changes in new push.